backend/Dockerfile

```
@@ -1,25 +1,23 @@
+RUN bun run build
 
-FROM node:18-alpine AS runner
+FROM oven/bun:1-alpine AS runner
 WORKDIR /usr/app
 ARG APP_ENV
 COPY --from=builder /app/build ./build
 COPY ./migrations ./build/migrations
 COPY package.json ./
 COPY --from=builder /app/config.json.default /usr/app/config.json
-RUN npm install
 USER root
```

Hadolint (2.14.0)

[warning] Last USER should not be root

(DL3002)